apps/omg_api/lib/ethereum_event_listener/core.ex

```diff
@@ -61,7 +61,7 @@ defmodule OMG.API.EthereumEventListener.Core do
         next_event_height_lower_bound: next_event_height_upper_bound + 1
     }
 
-    db_updates = [{:put, update_key, next_event_height_upper_bound}]
+    db_updates = [{:put, update_key, next_sync_height}]
```

Comment:
When we set last height to next_sync_height, update it in db and process crashes, then after restart we can get
next_event_height_lower_bound = next_sync_height. Then next_event_height_upper_bound < next_event_height_lower_bound. What happens then? Even if that works and Exthereumex does not crash, in another round we process the same events twice.

Reply:
Can EEL at the moment of writing to DB have events processed at the height < ethereum_height - finality_margin?

Reply:
Fixed, added test emulating service restart